ansible/roles/visual-studio/tasks/partials/vs2015.yml

```diff
@@ -8,7 +8,7 @@
   win_get_url:
     url: 'https://go.microsoft.com/fwlink/?LinkId=532606'
     dest: 'C:\TEMP\vs2015_community.exe'
-    force: no
+    force: false
```

Comment:
Indentation updates and `---` at the top sound great to me, but text should not change. Take this line for example, the Ansible documentation at https://docs.ansible.com/ansible/latest/modules/win_get_url_module.html clearly shows that the choices are `yes` and `no`.

Reply:
so `yes` and `no` are always interchangeable with `True` and `False` and ansible is pushing for defaulting to `True`/`False` in all newer versions. I'm open to either but I feel that we should agree on a format to use